src/tsung_controller/ts_config.erl

```diff
@@ -503,7 +503,13 @@ parse(_Element = #xmlElement{name='if', attributes=Attrs,content=Content},
     NewConf = lists:foldl(fun parse/2, Conf#config{curid=Id+1}, Content),
     NewId = NewConf#config.curid,
     ?LOGF("endif in session ~p as id ~p",[CurS#session.id,NewId+1],?INFO),
-    InitialAction = {ctrl_struct, {if_start, Rel, VarName, list_to_binary(Value) , NewId+1}},
+    SubstitutionFlag = case string:str(Value, "%%") of
```

Comment:
we should use ```re:run(Value, "%%.+%%")``` instead if think

Reply:
Okay. I thought `string:str` because that's recommended for performance reasons, but since we are only parsing the configuration, that should not be a issue to begin with.

I'll change this.

Do you have anything else on the new approach?